### PR TITLE
Feature/standalone docker

### PR DESCRIPTION
**Pull request by VargaJoe**

### CODE REVIEW

File: src/WebApps/SnWebApplication.Api.InMem.TokenAuth/Startup.cs [hunk 1]
```diff
@@ -56,9 +72,9 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
+                IdentityModelEventSource.ShowPII = true;
             }
-
-            app.UseHttpsRedirection();
+            //app.UseHttpsRedirection();
```
Comment (tusmester) [inline src/WebApps/SnWebApplication.Api.InMem.TokenAuth/Startup.cs:77]: Should we remove this? This is required in other cases, only causes problems in a docker environment, right? Maybe add https redirection based on a config?

File: src/WebApps/SnWebApplication.Api.InMem.TokenAuth/appsettings.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
   "sensenet": {
     "authentication": {
       "authority": "https://localhost:44311",
+      "metadatahost": "http://SenseNet.IdentityServer4.Web",
```
Comment (tusmester) [inline src/WebApps/SnWebApplication.Api.InMem.TokenAuth/appsettings.json:23]: When do we need this? Only in a docker env, or always? Maybe the default should be an empty string.

**Review by tusmester — CHANGES_REQUESTED** [review]

File: src/WebApps/SnWebApplication.Api.InMem.TokenAuth/SnWebApplication.Api.InMem.TokenAuth.csproj [hunk 3]
```diff
@@ -2,10 +2,15 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <UserSecretsId>5fc5a875-383a-4d51-ae31-06162f7c97f4</UserSecretsId>
+    <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+    <DockerfileContext>..\..</DockerfileContext>
+    <DockerfileRunArguments>--network sensenet --name SnWebApplication.Api.InMem.TokenAuth</DockerfileRunArguments>
```
Comment (tusmester) [inline src/WebApps/SnWebApplication.Api.InMem.TokenAuth/SnWebApplication.Api.InMem.TokenAuth.csproj:8]: Use a unified app name that is the same across all our web apps.

File: src/WebApps/SnWebApplication.Api.InMem.TokenAuth/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+FROM mcr.microsoft.com/dotnet/aspnet:3.1 AS base
+WORKDIR /app
+EXPOSE 80
+EXPOSE 443
+
+FROM mcr.microsoft.com/dotnet/sdk:3.1 AS build
+WORKDIR /src
+COPY ["WebApps/SnWebApplication.Api.InMem.TokenAuth/SnWebApplication.Api.InMem.TokenAuth.csproj", "WebApps/SnWebApplication.Api.InMem.TokenAuth/"]
+COPY ["OData/SenseNet.OData.csproj", "OData/"]
```
Comment (tusmester) [inline src/WebApps/SnWebApplication.Api.InMem.TokenAuth/Dockerfile:11]: Do we need to copy individual csproj files?

File: src/WebApps/SnWebApplication.Api.InMem.TokenAuth/Startup.cs (hunk 1 above)
Comment (VargaJoe) [reply src/WebApps/SnWebApplication.Api.InMem.TokenAuth/Startup.cs:77]: yes, services may communicate via http between containers directly instead public urls

File: src/WebApps/SnWebApplication.Api.InMem.TokenAuth/appsettings.json (hunk 2 above)
Comment (VargaJoe) [reply src/WebApps/SnWebApplication.Api.InMem.TokenAuth/appsettings.json:23]: containers have to use different channels when communicate directly

File: src/WebApps/SnWebApplication.Api.InMem.TokenAuth/Dockerfile (hunk 4 above)
Comment (VargaJoe) [reply src/WebApps/SnWebApplication.Api.InMem.TokenAuth/Dockerfile]: don't have to, vs creates in this form by default, I'll change this

File: src/WebApps/SnWebApplication.Api.InMem.TokenAuth/appsettings.json (hunk 2 above)
Comment (tusmester) [reply src/WebApps/SnWebApplication.Api.InMem.TokenAuth/appsettings.json]: we can probably move this to launchsettings for Docker

**Review by tusmester — APPROVED** [review]

**Pull request merged**